src/Common/SystemLogBase.cpp

```diff
@@ -262,8 +264,10 @@ void SystemLogBase<LogElement>::notifyFlush(bool force) { queue->notifyFlush(for
 
 #define INSTANTIATE_SYSTEM_LOG_BASE(ELEMENT) template class SystemLogBase<ELEMENT>;
 SYSTEM_LOG_ELEMENTS(INSTANTIATE_SYSTEM_LOG_BASE)
+SYSTEM_PERIODIC_LOG_ELEMENTS(INSTANTIATE_SYSTEM_LOG_BASE)
```

Comment:
I moved this here to avoid adding the periodic system log names both to `SYSTEM_LOG_ELEMENTS` and `SYSTEM_PERIODIC_LOG_ELEMENTS`. This way, there's only one define.